test/test.py

```diff
@@ -232,6 +232,8 @@ def test_specific_version_full(self):
     run_emsdk('activate sdk-tag-1.38.33-64bit')
 
   def test_binaryen_from_source(self):
+    if MACOS:
+      self.skipTest("https://github.com/WebAssembly/binaryen/issues/4299")
```

Comment:
I don't really see how this is better than just committing #924. But I don't feel strongly about it.

Reply:
The main difference is that all the tools that emskd builds will still be 10.11 compatible.   

if you try to build binaren latest or tot form source you will get a build error.. which is better than silently building a less-compatible version of binaryen perhaps?

Reply:
Oh I see, if you try to build older builds they will still work and be compatible. makes sense.